float64 command
---
bool success
string error_msg